react/features/toolbox/components/web/Toolbox.js

```diff
@@ -957,6 +959,9 @@ class Toolbox extends Component<Props, State> {
             t
         } = this.props;
 
+        const localParticipant = getParticipantById(this.props._localParticipantID);
```

Comment:
this functions has different API https://github.com/aeternity/jitsi-meet/blob/master/react/features/base/participants/functions.js#L102

I guess you need `getParticipantById(APP.store.getState(), this.props._localParticipantID)`

Reply:
Its working in the similar way, with APP.store.getState() or without.

Reply:
```js
const localParticipant = getParticipantById(this.props._localParticipantID);
console.log({ localParticipant });
// {localParticipant: undefined}
```

Reply:
so, no..

Reply:
but with `APP.store.getState()` as a first arg we get the data. also you can pass a selector as that param. but `APP.store.getState()` is totally fine

Reply:
I think @shapkarin is correct here. @DeeJayElly can you double check and try applying the suggested changes?

Reply:
Added